acctz/acctz.proto

```diff
@@ -210,10 +206,6 @@ message RecordResponse {
 
   // Authentication related details
   AuthDetail authen = 7;
-
-  // Optional repeated task_id that represent tasks that were used to
-  // accomplish the request on the system.
-  repeated string task_ids = 32;
```

Comment:
My understanding off this 2-line comment would be that this could represent any system defined method to "trace" a call.  Could you elaborate on it's intent or shall we mark for removal and define what this might mean later?

Reply:
I think these are intended to be IDs that could be used to correlate processes etc with an RPC/cmd/etc.   I believe that some systems include such IDs in syslog msgs.  But, I was not around when this was created, so I am speculating.

Reply:
Similar to other cases, if there is no specification or concrete intent considering how such would be implemented consistently, etc..  then I would suggest it's removal to be reintroduced at a later point in time should there be a need and detailed specification.

Reply:
Marcus & Morrow should comment on this; they are more likely to know the intention.

Reply:
Today we get this sort of notation in tac+ accounting logs, it's what stitches together all commands/etc sent over the life of a user's "session".

That's the intent of this field.
That may not matter for gRPRC things - 'connect send config, disconnect' - session-id is same as singular record.
It does matter for user-that-sshs sessions though. (I think it does, still matter)

Reply:
@morrowc You mentioned this field matters for SSH access, it can be the pid of the sshd instance serving the connection ?

Reply:
Could be ssh-pid, sure!
I think I figured the universal answer: "uuid" would work here too.
but yup, sshd pid seems ok too.